packages/core/assemblyManager/assemblyManager.ts

```diff
@@ -30,7 +29,10 @@ export default function assemblyManagerFactory(
       },
 
       get assemblyList() {
-        return getParent(self).jbrowse.assemblies.slice()
+        return [
+          ...getParent(self).jbrowse.assemblies.slice(),
+          ...(getParent(self).session.sessionAssemblies?.slice() || []),
```

Comment:
do you actually need to use slice if you are spreading also?

Reply:
apparently not:)